### PR TITLE
Find a Rep 2.0 - copy/styling/a11y updates + unit test coverage

### PR DESCRIPTION
**Pull request by cosu419**
This PR contains some minor styling/component updates, as well as update unit test coverage. 

## Summary

- Copy updates
- Styling updates
- Scroll/focus updates per a11y recommendations
- Removed unused methods
- Reducer unit tests
- Report Modal tests
- Search Result tests

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/76061
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/75813
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/75874
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/74720

## Testing done
Improved test coverage for reducers and new component features. 

## Screenshots
image

## What areas of the site does it impact?
Staging (app is not enabled in production)

### CODE REVIEW

File: src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ describe('SearchResultsHeader', () => {
         .text()
         .replace(/[^A-Za-z0-9" ]/g, ' '),
     ).to.equal(
-      'No results found for "Accredited attorney" within "50 miles" of "11111"',
+      'No results found for "Accredited attorney" within "50 miles" of "11111" sorted by "Distance  closest to farthest "',
```
Comment (holdenhinkle) [inline src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx:44]: Is there an extra space between 'Distance' and 'closest'?

File: src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx [hunk 2]
```diff
@@ -78,7 +80,7 @@ describe('SearchResultsHeader', () => {
         .text()
         .replace(/[^A-Za-z0-9" ]/g, ' '),
     ).to.equal(
-      'No results found for "Accredited attorney" within "50 miles" of "new york"',
+      'No results found for "Accredited attorney" within "50 miles" of "new york" sorted by "Distance  closest to farthest "',
```
Comment (holdenhinkle) [inline src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx:83]: Is there an extra space between 'Distance' and 'closest'?

File: src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx [hunk 3]
```diff
@@ -104,7 +107,7 @@ describe('SearchResultsHeader', () => {
         .text()
         .replace(/[^A-Za-z0-9" ]/g, ' '),
     ).to.equal(
-      'Showing 1 result for "Accredited attorney" within "50 miles" of "new york"',
+      'Showing 1 result for "Accredited attorney" within "50 miles" of "new york" sorted by "Distance  closest to farthest "',
```
Comment (holdenhinkle) [inline src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx:110]: Is there an extra space between 'Distance' and 'closest'?

File: src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx [hunk 4]
```diff
@@ -173,13 +177,14 @@ describe('SearchResultsHeader', () => {
...
           searchArea: '50',
+          sortType: 'distance_asc',
         }}
         pagination={{ totalEntries: 12, currentPage: 1, totalPages: 2 }}
       />,
     );
 
     const expectedString =
-      'Showing1-10of12resultsfor"Accreditedattorney"within"50miles"of"newyork"';
+      'Showing1-10of12resultsfor"Accreditedattorney"within"50miles"of"newyork" sorted by "Distance  (closest to farthest)"';
```
Comment (holdenhinkle) [inline src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx:187]: Is there an extra space between 'Distance' and 'closest'?

File: src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx [hunk 5]
```diff
@@ -278,7 +286,7 @@ describe('SearchResultsHeader', () => {
         .text()
         .replace(/[^A-Za-z0-9" ]/g, ' '),
     ).to.equal(
-      'No results found for "Accredited claims agent" within "50 miles" of "new york"',
+      'No results found for "Accredited claims agent" within "50 miles" of "new york" sorted by "Distance  closest to farthest "',
```
Comment (holdenhinkle) [inline src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx:289]: Is there an extra space between 'Distance' and 'closest'?

**Review by holdenhinkle — CHANGES_REQUESTED** [review]
Nice tests!

There's 1 small issue with a string that shows up in your tests a number of times.

**Review by holdenhinkle — APPROVED** [review]
Ignoring that spacing issue for now, so this is ready for staging review.

Again, nice tests.

**Review by micahchiang — APPROVED** [review]

File: src/applications/representative-search/components/results/ReportModal.jsx [hunk 6]
```diff
@@ -122,9 +123,43 @@ const ReportModal = ({
...
+        <h2
+          className="report-modal-header"
+          style={{ fontSize: 20, marginTop: 10 }}
+        >
+          Report outdated information for {representativeName}
+        </h2>
         {someItemsReported && (
           <>
-            <h4>You reported this information</h4>
+            <h3 style={{ fontSize: 17, marginTop: 20 }}>
```
[the remark this answers is not in the document]
Comment (cosu419) [reply src/applications/representative-search/components/results/ReportModal.jsx:160]: No solid reasoning - created a [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76732) to remove the handful of inline styling instances throughout the app. Thanks!

**Pull request merged**